hooks/pre/19-devlog.rb

```diff
@@ -0,0 +1,11 @@
+# check to see if /dev/log directory exists
+
+dir_present = File.exist?('/dev/log')
```

Comment:
Maybe move this to the elsif statement so skipping the check really does skip everything related?